helm/mds/templates/natss-channel.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: messaging.knative.dev/v1alpha1
+kind: NatssChannel
+metadata:
+  name: natss
```

Comment:
Maybe combine this with `natss-broker.yaml` and just call it `natss.yaml`?  Also, this doesn't need a namespace?

Reply:
Picking this up in James' feature branch